internal/services/postgres/validate/flexible_server_storage_tier.go

```diff
@@ -88,3 +91,15 @@ func InitializeFlexibleServerStorageTierDefaults() map[int]StorageTiers {
 
 	return storageTiersMappings
 }
+
+// CompareAzureManagedDiskPerformancef returns an bool comparing two AzureManagedDiskPerformanceTiers.
+// The result will be false if disk == compareDisk, false if disk < compareDisk, and true if disk > compareDisk.
+func CompareAzureManagedDiskPerformance(disk, compareDisk servers.AzureManagedDiskPerformanceTiers) bool {
```

Comment:
I assume that we don't need it anymore if using "IsExplicitlyNullInConfig" to fix the issue.